exercises/operations/operations1.cairo

```diff
@@ -0,0 +1,22 @@
+// Felts supports basic math operations.
```

Comment:
It would also be a good idea to include an exercise with integer types, who support more operators then felts (i.e. `%`)

Reply:
added